### DISCUSSION TITLE
3D arc with 2 points and a radius

### DISCUSSION THREAD

**Discussion by danieltameer** [mozman/ezdxf#1034] [General]
I want to construct an arc in 3D, with 2 known 3D coordinates and a known bend radius value. I can only find how to do this in 2D but cannot figure out how to do this in 3D.

Would appreciate any advice, thanks

**Comment by mozman** [comment]
Hi!

There are an infinite number of solutions with these boundary conditions. All circles with the given radius that pass the given 2 points.

![image](https://github.com/mozman/ezdxf/assets/4230966/9e84a2e5-7a69-42ad-83ee-d09b4cd0d465)
